templates/aem-tools/deploy-artifact.sh.epp

```diff
@@ -51,4 +53,14 @@ FACTER_aem_username="${aem_username}" \
   --hiera_config conf/hiera.yaml \
   --execute "include aem_curator::action_deploy_artifact"
 
+  echo "Testing if package is installed using InSpec..."
+  cd "<%= $base_dir %>/aem-aws-stack-provisioner/modules/aem_curator/inspec/profiles/inspec-aem"
```

Comment:
Curator shouldn't need to know about stack provisioner (note that it's a platform specific location, i.e. *aws* ).
If it's necessary to navigate to a certain location, it's the provisioner's responsibility to make the profiles available at a location relative to the shell scripts.